doc/cephfs/nfs.rst

```diff
@@ -56,6 +56,18 @@ in the sample conf. There are options to do the following:
 - enable read delegations (need at least v13.0.1 ``libcephfs2`` package
   and v2.6.0 stable ``nfs-ganesha`` and ``nfs-ganesha-ceph`` packages)
 
+.. important::
+
+   Under certain conditions, NFS access using CephFS FSAL fails. This causes an
```

Comment:
I'm not 100% positive, but I think `the CephFS FSAL` might be better.

Reply:
Accepted; integrated.